src/scripts/visualizations/line.js

```diff
@@ -139,7 +139,7 @@
         }
 
         function renderMarkers() {
-            var animationDelay = options.line.marker.animationDelay;
+            var animationDelay = (options.line.marker.animationDelay == null) ? duration : options.line.marker.animationDelay;
```

Comment:
i'm surprised jshint doesn't complain if you use `==` like that; does your editor not complain?